src/utils/phone.ts

```
@@ -3,22 +3,26 @@ export const formatPhoneNumber = (phone: string) => {
   const digits = phone.replace(/\D/g, "").slice(0, 11);
   if (!digits) return "";
 
-  // 입력 중 포맷: 3-4-4 기준으로 점진적 하이픈 삽입
-  if (digits.length <= 3) return digits;
-  if (digits.length <= 7) {
+  const digits = phone.replace(/\D/g, ""); // 숫자 이외 제거
```

Biome (2.3.13)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)